ext/simplexml/simplexml.c

```diff
@@ -975,10 +975,9 @@ static int sxe_prop_is_empty(zend_object *object) /* {{{ */
 	php_sxe_object  *sxe;
 	xmlNodePtr       node;
 	xmlAttrPtr       attr;
-	zval             iter_data;
 	int              test;
 	int              is_empty;
```

Comment:
Aside: ``is_empty`` also looks to be a boolean value that is returned. So maybe a follow up is changing the signature of this static function? (or is it an object handler and thus cannot be changed atm)

Reply:
We can change it without problems. I'm gonna do a scope cleanup pass sometime soon anyway because having everything on top makes it more difficult to reason what's used where and what's safe to change. I'll take that in that PR.